Applications/Spire/Source/Ui/NavigationView.cpp

```diff
@@ -71,7 +81,7 @@ class NavigationView::NavigationTab : public QWidget {
       auto container = new LabelContainer(std::move(label));
       container->setSizePolicy(QSizePolicy::Expanding, QSizePolicy::Expanding);
       layers->add(container);
-      auto tab = new Tab();
+      auto tab = new Box(nullptr);
```

Comment:
The ```Tab``` and ```SelectLine``` were excellent, by all means keep them. You could even inherit from ```Box``` for both of them.